### ISSUE TITLE
--help exits non-zero

### ISSUE THREAD

**Issue by allenap** [rtfeldman/node-test-runner#359]
I think it's usual that a program would exit zero (success) after being invoked with `--help`. I tried it out with `elm`, `ls`, `find`, and a few others – they all exit zero.

I use `elm-test --help` as a "liveness" test – and also to give it an opportunity to download binaries – so exiting zero would be useful to make my process more robust.

I think the following change will do it:

```diff
--- lib/elm-test.js
+++ lib/elm-test.js
@@ -125,7 +125,7 @@ if (args.help) {
     '[--watch] # Run tests on file changes',
   ].forEach(printUsage);

-  process.exit(1);
+  process.exit(0);
 }

 if (args.version) {
```

**Comment by harrysarson** [comment]
@rtfeldman it has occurred to me that this is probably a semver major change. I have no idea how elm-test should follow semver though?

**Comment by harrysarson** [comment]
I guess this change will be released with the 0.19.1 version of elm test which can reasonably containing breaking changes as `elm-test` is in 0.x versioning.
